changelog.md

```diff
@@ -20,10 +20,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - \[Demo\] Customization radio buttons has been replaced by chips ([#169](https://github.com/Orange-OpenSource/ods-android/issues/169))
 - \[Demo\] The number of items in bottom navigation can now be changed using plus and minus buttons ([#172](https://github.com/Orange-OpenSource/ods-android/issues/172))
 - \[Demo\] Update lists component UI ([#141](https://github.com/Orange-OpenSource/ods-android/issues/141))
+- \[Lib\] Change `OdsSlider` signature: `leftIconRes` and `rightIconRes` have been replaced respectively by `leftIcon` and `rightIcon` which are `Painter` typed
```

Comment:
Same comment as previous PR, I think we should add the related issue at the end of the line. Although this modification was not the purpose of the issue, it will allow us to track in which PR the code was modified.